qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -65,32 +57,7 @@
         "PMD.TooManyMethods", "PMD.AvoidDuplicateLiterals", "PMD.GodClass"
     }
 )
-final class CheckstyleValidatorTest {
-
-    /**
-     * Name of property to set to change location of the license.
-     */
-    private static final String LICENSE_PROP = "license";
-
-    /**
-     * Directory with classes.
-     */
-    private static final String DIRECTORY = "src/main/java/foo";
-
-    /**
-     * License text.
-     */
-    private static final String LICENSE = "Hello.";
-
-    /**
-     * Rule for testing.
-     */
-    private License rule;
-
-    @BeforeEach
-    public void setRule() {
-        this.rule = new License();
-    }
+final class CheckstyleValidatorTest extends CheckstyleTestBase {
```

Comment:
@IlyaPechersky inheritance, in general, is a bad practice in OOP: https://www.yegor256.com/2016/09/13/inheritance-is-procedural.html why do you need it here? just one class will your base class. why?

Reply:
@yegor256 Thanks for the article! Replaced it with a utility class